public/landing.html

```diff
@@ -28,6 +28,282 @@
   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,300italic,400italic,600italic,700italic,800,800italic' rel='stylesheet' type='text/css'>
 </head>
 <body>
-  <h1>Good luck!</h1>
+  <nav class="navbar navbar-default">
```

Comment:
I believe I added the `navbar-fixed-top` class also, in order to get it to stick to the top. This should take care of the extra whitespace that you see at the top of your page.

http://getbootstrap.com/components/#navbar-fixed-top